packages/app/src/app/overmind/effects/api/apiFactory.ts

```diff
@@ -14,6 +14,11 @@ const getBaseApi = (path: string, useRoot: boolean = false) => {
     return API_ROOT;
   }
 
+  // Special case for /auth/workos requests which are not on /api/v1
```

Comment:
Why is it not on /api/v1? On premise?

Reply:
the /auth/workos routes are not in /api/v1, some of them are json endpoints, others are html